src/classes/route.py

```diff
@@ -2,10 +2,13 @@
 from .edge import Edge, get_length  
 
 class Route():
+    id: int                             = None           # vehicle identifier  
```

Comment:
I think the spacing, while clean, will be harder to keep consistent across all the files we have. Also perhaps id should be generated by the class itself vs being a parameter that is passed in. A global iterator should suffice (every new vehicle increments the id by one), or perhaps some built-in identifier. I will leave that decision up to you.